File: CustomerEmailWebAPI/Models/CustomerEmail.cs
namespace CustomerEmailProgram.Models
{
    public class CustomerEmail
    {
        public string CustomerName { get; set; }
        public Guid CustomerId { get; set; }
        public string Message { get; set; }        
        public DateTime CreatedDate { get; set; }
        public DateTime LastUpdatedDate { get; set; }
    }
}

File: CustomerEmailAngularWebPage/src/app/models/customer-email.ts
export class CustomerEmail {
    customerName: string = "";
    customerId: string = "";
    message: string = "";
    emailId: string = "";
    isDraft: boolean = false;
    createdDate: Date = new Date();
    lastUpdatedDate: Date = new Date();
    constructor(customerName: string, customerId: string, message: string, emailId: string,
        isDraft: boolean, createdDate: Date, lastUpdatedDate: Date
    ) {
        this.customerName = customerName;
        this.customerId = customerId;
        this.message = message;
        this.emailId = emailId;
        this.isDraft = isDraft;
        this.createdDate = createdDate;
        this.lastUpdatedDate = lastUpdatedDate;
    }
}